docs/advanced/document-store/milvus.md

```diff
@@ -0,0 +1,512 @@
+(milvus)=
+# Milvus
+
+One can use [Milvus](https://milvus.io/) as the Document store for DocumentArray. It is useful when one wants to have faster Document retrieval on embeddings, i.e. `.match()`, `.find()`.
+
+````{tip}
+This feature requires `pymilvus`. You can install it via `pip install "docarray[milvus]".` 
+````
+
+## Usage
+
+### Start Milvus service
+
+To use Milvus as the storage backend, you need a running Milvus server. You can use the following `docker-compose.yml`
+to start a Milvus server:
+
+`````{dropdown} docker-compose.yml
+
+```yaml
+version: '3.5'
+
+services:
+  etcd:
+    container_name: milvus-etcd
+    image: quay.io/coreos/etcd:v3.5.0
+    environment:
+      - ETCD_AUTO_COMPACTION_MODE=revision
+      - ETCD_AUTO_COMPACTION_RETENTION=1000
+      - ETCD_QUOTA_BACKEND_BYTES=4294967296
+      - ETCD_SNAPSHOT_COUNT=50000
+    volumes:
+      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/etcd:/etcd
+    command: etcd -advertise-client-urls=http://127.0.0.1:2379 -listen-client-urls http://0.0.0.0:2379 --data-dir /etcd
+
+  minio:
```

Comment:
I'm afraid this  can get stale at some point (although we are actually pinning container versions).
It would be really great if we find this yaml in the official versioned documentation and point to it